Rules/ProvideCommentHelp.cs

```diff
@@ -47,10 +47,17 @@ public override AstVisitAction VisitFunctionDefinition(FunctionDefinitionAst fun
                 return AstVisitAction.SkipChildren;
             }
 
-            if (funcAst.GetHelpContent() == null)
+            if (!string.Equals(funcAst.Name, "Get-TargetResource", StringComparison.OrdinalIgnoreCase) && !string.Equals(funcAst.Name, "Set-TargetResource", StringComparison.OrdinalIgnoreCase) &&
```

Comment:
Hemant said DSC engine does not understand aliases so checks for Get/Set/Test should be fine.